.evergreen/scripts/run-ocsp-test.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
```

Comment:
A general thought, what if we had a wrapper bash script that we run that invokes the env.sh file, uses `set -eu`, and then invokes the target script.

Reply:
I like that idea. More standardization is always good.